### PR TITLE
fix(context menu): context menu scroll bars

### PR DESCRIPTION
**Pull request by fschade**
in some cases it could happen that the screen height is not
enough to display all context menu items. This is fixed by
calculating available screen estate and makes the container scrollable

fix: https://github.com/owncloud/web/issues/5845

### CODE REVIEW

File: changelog/unreleased/bugfix-context-menu-scroll [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Bugfix: Prevent hover style on footer of <oc-table>
```
Comment (pascalwengerter) [inline changelog/unreleased/bugfix-context-menu-scroll:1]:
```suggestion
Bugfix: Prevent contextmenu display issues within OcTableFiles
```

File: changelog/unreleased/bugfix-context-menu-scroll [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+
+Context menu for files table now detects if available space is enough to show all items. If not,
+it automatically calculates the height and adds scroll bars to the menu.
+
+https://github.com/owncloud/web/pull/5871
```
Comment (pascalwengerter) [inline changelog/unreleased/bugfix-context-menu-scroll:6]:
```suggestion
https://github.com/owncloud/owncloud-design-system/pull/1691
https://github.com/owncloud/web/issues/5845
```

File: src/components/table/OcTableFiles.vue [hunk 3]
```diff
@@ -681,391 +710,4 @@ export default {
   }
 </script>
 ```
-
-## Shared with others files table
```
Comment (pascalwengerter) [inline src/components/table/OcTableFiles.vue:685]: Why delete all the examples? I found them quite useful in the past:)

**Review by pascalwengerter — CHANGES_REQUESTED** [review]

File: src/components/table/OcTableFiles.vue (hunk 3 above)
Comment (fschade) [reply src/components/table/OcTableFiles.vue:685]: because i'm an idiot, i bring it back. Thanks for pointing out

**Review by pascalwengerter — APPROVED** [review]
LGTM, please squash-merge;)

**Pull request merged**